doc/uart_configuration.rst

```diff
@@ -0,0 +1,343 @@
+.. _uart_configuration:
+
+UART Configuration
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section describes the UART configuration for the |SM| application.
+
+UART interface
+**************
+
+A typical Serial Modem application configuration consists of a host processor connected to an nRF91 Series SiP via a UART interface.
+The UART connection serves as the physical transport layer, carrying both AT commands and PPP data frames for network communication.
+
+.. figure:: images/samples_overview.svg
```

Comment:
```suggestion
.. figure:: images/cmux-wakeup-timing.svg
```

Reply:
I believe you mean `images/cellular-serial-setup.svg` which I switched to now.